VultisigApp/VultisigApp/Services/BalanceService.swift

```diff
@@ -20,78 +20,92 @@ class BalanceService {
     private let kuji = KujiraService.shared
     private let maya = MayachainService.shared
     private let dot = PolkadotService.shared
-
+    
+    private var cache: ThreadSafeDictionary<String, (data: Bool, timestamp: Date)> = ThreadSafeDictionary()
```

Comment:
We are not using the cache, can you please just remove it , use a date instead?

Reply:
I will add a var then.

Reply:
nevermind , I misunderstand it